### ISSUE TITLE
Staging broken due to taxonomy unit not loading

### ISSUE THREAD

**Issue by alexgarel** [openfoodfacts/openfoodfacts-server#10012]
modperl_error_log
```log
[Mon Mar 25 09:38:30.615916 2024] [perl:error] [pid 17] Can't load Perl file: /opt/product-opener/lib/startup_apache2.pl for server openfoodfacts.net:0, exiting...
```

log4perl.log
```log
[2024/03/25 09:38:30] [11] /opt/product-opener/lib/ProductOpener/Tags.pm 2484 ProductOpener.Tags {} Could not load taxonomy units - dying
```

**Comment by raphael0202** [comment]
Based on graphs on monitoring.openfoodfacts.org, it looks like [this commit](https://github.com/openfoodfacts/openfoodfacts-server/commit/4edbfd00779bfd153d8047f6df76338bd816e3d8) is the faulty one.

**Comment by alexgarel** [comment]
Fixed by #10020
